portality/static/doaj/doajArticles.xsd

```diff
@@ -97,8 +97,6 @@
        <xs:complexType>
         <xs:sequence>
          <xs:element name="name" type="xs:string" />
-         <xs:element name="email" type="xs:string"
```

Comment:
This may need to be communicated in advance of the release. Potentially a temporally restricted note on the XML upload page too?